label_studio/data_manager/managers.py

```diff
@@ -496,12 +510,60 @@ def __init__(self, expression, distinct=False, output_field=None, **extra):
 
 
 def annotate_completed_at(queryset):
+    if flag_set('fflag_feat_optic_161_project_settings_for_low_agreement_threshold_score_short'):
+        return annotated_completed_at_considering_agreement_threshold(queryset)
```

Comment:
basicly this is hidding from the rest of users, correct?

Reply:
Yes. It leaves the default behavior/queries in place if the user doesn't have the feature flag on